fastlane/lib/fastlane/actions/docs/upload_to_testflight.md

```diff
@@ -173,7 +173,13 @@ fastlane pilot export
 
 ### Import testers
 
-Add external testers from a CSV file. Sample CSV file available [here](https://itunesconnect.apple.com/itc/docs/tester_import.csv).
+Add external testers from a CSV file. Create a file called `tester_import.csv` and fill it with the following format:
```

Comment:
>Create a file called `tester_import.csv` and

Can we change this to "Create a CSV file (ex: `tester.csv`) and"? Just wanted to make sure its clear that the file doesn't _need_ to be called `tester_import.csv`